otherlibs/win32unix/putenv.c

```diff
@@ -0,0 +1,46 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           */
+/*                                                                        */
```

Comment:
Add a line with the name of the new author.

Reply:
Thanks, fixed.